jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py

```diff
@@ -42,8 +44,12 @@ def __init__(self):
         self.use_twitter = rospy.get_param("~use_twitter", True)
         self.use_google_chat = rospy.get_param(
             "~use_google_chat", _enable_google_chat)
+        self.send_every_transition = rospy.get_param(
+            "~send_every_transition", False)
```

Comment:
if you have dynamic recomfigure, you dont need get_param

Reply:
Thank you for notifying.
I updated in [8cd3adb](https://github.com/jsk-ros-pkg/jsk_robot/pull/1760/commits/8cd3adb6b0b5672a686b9c656a53214d38800168)